Core/Business/CMS/Core_Business_CMS_CMSRepository.php

```diff
@@ -30,7 +30,7 @@ class Core_Business_CMS_CMSRepository extends Core_Foundation_Database_EntityRep
      * Return CMSRepository lang associative table name
      * @return string
      */
-    private function getLanguageTableNameWithPrefix()
+    final private function getLanguageTableNameWithPrefix()
```

Comment:
PHP OOP rule n°123234: ALL private methods will be final. This avoid many problems.